tests/test_security.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""
+Test the authentication and permission of Course Access Groups.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import pytest
+from rest_framework.authentication import TokenAuthentication
+from openedx.core.lib.api.authentication import OAuth2Authentication
+from course_access_groups.security import (
+    CommonAuthMixin,
+)
+
+
+class TestCommonAuthMixin(object):
```

Comment:
Is there really a need for a test class here? Wouldn't a test function be good enough? I'm really not seeing where the test class in this case is adding any value

Reply:
Good question. Yes, because I'm planning to expand it in the future.